src/test/java/de/tum/in/www1/artemis/exercise/programmingexercise/ProgrammingAssessmentIntegrationTest.java

```diff
@@ -676,6 +678,48 @@ void cancelAssessmentOfOtherTutorAsInstructor() throws Exception {
         cancelAssessment(HttpStatus.OK);
     }
 
+    @Test
+    @WithMockUser(username = TEST_PREFIX + "tutor1", roles = "TA")
+    void testSaveAssessmentNote() throws Exception {
+        AssessmentNote assessmentNote = new AssessmentNote();
+        assertThat(assessmentNote.getCreatedDate()).isNotNull();
+
+        assessmentNote.setNote("note");
+        manualResult.setAssessmentNote(assessmentNote);
+
+        User user = userUtilService.getUserByLogin(TEST_PREFIX + "tutor1");
```

Comment:
Maybe simply use userRepository.getUser(), I think that retrieves the currently logged-in user, which in this case should be the user from the @WithMockUser annotation.